code-review-trainer-app/yarn.lock

```diff
@@ -38,12 +38,12 @@
 
 "@babel/compat-data@^7.27.2":
   version "7.28.0"
-  resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.28.0.tgz#9fc6fd58c2a6a15243cd13983224968392070790"
+  resolved "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.28.0.tgz"
```

Comment:
It looks like you changed this file way more than expected. Shouldn't we continue using registry.yarnpkg.com?